docs/Course Wiki/ECE Course Offerings/ECE374B.md

```diff
@@ -0,0 +1,66 @@
+# ECE374B
+## Introduction to Algorithms and Models of Computation
+
+ECE374B is one of two **independent** sections of CS/ECE374, Introduction to Algorithms and Models of Computation. It is unoficially the 'ECE' version of 374. It is considered by some to be easier than the A version.
+
+## Content Covered
+
+#### Theory of Computation 
+- Regular and context-free languages.
+- Fooling sets.
+- Determinisitc and non-deterministic finite state automata.
+- Push down automata
+- Turing Machines
+- Computational Complexity
+#### Algorithms
+- Divide & Conquer
+- Graph Algorithms
+- Shortest Paths, MSTs
+- Dynamic Programming
+- Directed Acylic Graphs
+- Algorithmic Complexity Analysis
+#### Secret Third Thing
+- Reductions
+- Undecidability
+- NP, NP-Hardness, P vs NP
```

Comment:
I like this formatting we should use it for the other classes

Reply:
Can you add something about how the content flows from the beginning to the end (i.e. what topics in the second thing are used in the third thing)?

Remove periods at the end of each bullet point.